### PR TITLE
Clear up confusion about what kind of refinement is to be explored

### PR DESCRIPTION
**Pull request by katcosburn**

### CODE REVIEW

**Comment by bangerth** [comment]
Do you think you could add an entry to `doc/news/changes/minor` that explains your change?

File: doc/news/changes/minor/20190808KatherineCosburn [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The original documentation in step-2 under "possible extensions" states in the second paragraph: "Or, you could see how the sparsity pattern changes with more refinements." 
+
+I found this to be redundant and confusing given that, in the paragraph before it, we are asked to explore how the sparsity pattern changes when we use higher order elements in the distribute_dofs function (ie. p-refinement).
+
+I assumed that what was meant in this second paragraph was to explore how the pattern changes when we refine the mesh (ie. h-refinement) so I edited to make this more clear.
+
+(Katherine Cosburn, 2019/08/08)
```
Comment (masterleinad) [inline doc/news/changes/minor/20190808KatherineCosburn]: Could you try to be a little bit more concise in the changelog entry? We normally try to describe changes in one or two sentences.

File: doc/news/changes/minor/20190808KatherineCosburn [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The original documentation in step-2 under "possible extensions" states in the second paragraph: "Or, you could see how the sparsity pattern changes with more refinements." 
+
+I found this to be redundant and confusing given that, in the paragraph before it, we are asked to explore how the sparsity pattern changes when we use higher order elements in the distribute_dofs function (ie. p-refinement).
+
+I assumed that what was meant in this second paragraph was to explore how the pattern changes when we refine the mesh (ie. h-refinement) so I edited to make this more clear.
+
```
Comment (masterleinad) [inline doc/news/changes/minor/20190808KatherineCosburn]:
```suggestion
<br>
```

File: doc/news/changes/minor/20190808KatherineCosburn [hunk 3]
```diff
@@ -0,0 +1,7 @@
+The original documentation in step-2 under "possible extensions" states in the second paragraph: "Or, you could see how the sparsity pattern changes with more refinements."
```
Comment (masterleinad) [inline doc/news/changes/minor/20190808KatherineCosburn]:
```suggestion
Improved: The original documentation in step-2 under "possible extensions" states in the second paragraph: "Or, you could see how the sparsity pattern changes with more refinements." 
```

File: doc/news/changes/minor/20190808KatherineCosburn (hunk 1 above)
Comment (katcosburn) [reply doc/news/changes/minor/20190808KatherineCosburn]: Yes! Sorry, I hope my newest version suitably concise.

**Comment by drwells** [comment]
Thanks @katcosburn!

**Pull request merged**